src/components/ClimbDropdown.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-menu class="mr-3">
+    <template v-slot:activator="{ on }">
+      <v-btn
+        :color="states[route_state].color"
+        class="white--text"
+        v-on="on"
+      >
+        {{ states[route_state].abbrev }}
+      </v-btn>
+    </template>
+    <v-list>
+      <v-list-tile
+        v-for="state in orderedStates"
+        :key="state"
+        @click="route_state = state"
+      >
+        <v-list-tile-title>{{ states[state].name }}</v-list-tile-title>
+      </v-list-tile>
+    </v-list>
+  </v-menu>
+</template>
+
+<script>
+import { ClimbState } from './ClimbState.js'
+
+export default {
+  props: ['route_state'],
```

Comment:
```suggestion
  props: ['routeState'],
```

see https://vuejs.org/v2/guide/components-props.html. i think the convention is to use routeState here, but then it'll seemingly be auto-mangled to route-state when you pass a value in HTML.